tests/unitary/test_from_etherscan.py

```diff
@@ -0,0 +1,36 @@
+import boa, os
+
+crvusd = '0xf939E0A03FB07F59A73314E73794Be0E57ac1b4E'
+voting_agent = "0xE478de485ad2fe566d49342Cbd03E49ed7DB3356"
+
+
+def test_from_etherscan_abi():
+
+    boa.env.fork(os.environ["ALCHEMY_MAINNET_ENDPOINT"])
+    abi = boa.from_etherscan_abi(crvusd, api_key=os.environ["ETHERSCAN_API_KEY"])
+
+    return abi
```

Comment:
it's not a real test if there's no assertion (same for all tests here). For example:
```suggestion
    assert abi.totalSupply() > 0
```